src/gui/statusbar.cpp

```diff
@@ -224,6 +234,14 @@ void StatusBar::refreshStatusBar()
     updateConnectionStatus(sessionStatus);
     updateDHTNodesNumber(sessionStatus);
     updateSpeedLabels(sessionStatus);
+    updateFreeSpaceOnDisk();
+}
+
+void StatusBar::updateFreeSpaceOnDisk()
+{
+    QString defaultPath = BitTorrent::Session::instance()->defaultSavePath();
+    m_freeSpaceOnDiskLbl->setText(tr("Free space: %1").arg(Utils::Misc::friendlyUnit(Utils::Fs::freeDiskSpaceOnPath(defaultPath))));
+    m_freeSpaceOnDiskLbl->setToolTip(defaultPath);
```

Comment:
What about ?

setToolTip(tr("Free disk space in default save path:") + '\n' + defaultPath)